dateparser/date.py

```diff
@@ -254,6 +252,40 @@ def _is_valid_date_obj(self, date_obj):
         return True
 
 
+class DateData:
+    """
+    Class that represents the parsed data with useful information.
+    It can be accessed like a dict object.
+    """
+
+    def __init__(self, date_obj=None, period=None, locale=None, is_relative=None):
```

Comment:
I don’t see `is_relative` used anywhere. Maybe we should delay its inclusion into the class until we have a use for it.

Reply:
At least it's used by `search_dates`, but I didn't deep too much.

Reply:
You were right, I could remove that field after refactoring a part of the `search_dates` part